src/wayland/handlers/mod.rs

```diff
@@ -9,6 +9,7 @@ pub mod dmabuf;
 pub mod drm;
 pub mod drm_lease;
 pub mod fractional_scale;
+pub mod idle;
```

Comment:
Can you split this into two modules for `idle_notify` and `idle_inhibit` please?
I know they are related in how this implementation currently works, but they don't have to be.